parsers/ENTSOE.py

```diff
@@ -895,8 +895,8 @@ def fetch_production(zone_key, session=None, target_datetime=None,
 
 
 ZONE_KEY_AGGREGATES = {
-    'IT-SIC': ['IT-SIC', 'IT-PR'],
-    'IT-SO': ['IT-FO', 'IT-BR', 'IT-RO', 'IT-SO'],
+    'IT-SIC': ['IT-SIC'],
```

Comment:
this should be removed, and in zones.json we should change IT-SIC parser from `ENTSOE.fetch_production_aggregate` to `ENTSOE.fetch_production`